manifests/templates/file-host.yaml.in

```diff
@@ -89,7 +89,7 @@ metadata:
 spec:
   selector:
       cdi.kubevirt.io/testing: ""
-  type: ClusterIP
+  type: NodePort
```

Comment:
note - tests have to communicate with the file host to retrieve the files meta data